pkg/cloudprovider/vsphere/vapilogger.go

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vsphere
+
+import (
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/log"
+	"k8s.io/klog"
+)
+
+type klogBridge struct {
```

Comment:
Can you comment on why this is needed?

Reply:
The new GO SDK uses an own logger and logs by default a lot of stuff to the standard file descriptors. Therefore Martin has added a bridge to the logger used in the kubernetes cloud provider manager.